packages/bitcore-wallet-client/src/lib/credentials.ts

```diff
@@ -162,24 +163,10 @@ export class Credentials {
           purpose = '48';
           break;
       }
-
-      var coin = '0';
-      if (this.network != 'livenet' && this.coin !== 'eth') {
-        coin = '1';
-      } else if (this.coin == 'bch') {
-        if (this.use145forBCH) {
-          coin = '145';
-        } else {
-          coin = '0';
-        }
-      } else if (this.coin == 'btc') {
-        coin = '0';
-      } else if (this.coin == 'eth') {
-        coin = '60';
-      } else {
-        throw new Error('unknown coin: ' + this.coin);
+      if (!this.use145forBCH && this.coin === 'bch') {
```

Comment:
it dont think this is a good idea. sending the wrong coin just to get the right path can easily generate problems in future changes.